### DISCUSSION TITLE
Close menu if you click outside

### DISCUSSION THREAD

**Discussion by Zerotask** [saadeghi/daisyui#2293] [Ideas / feature requests]
I'm using a navbar with a menu, similar to this: https://daisyui.com/components/navbar/#navbar-with-menu-and-submenu

It would be great to close the submenu, if you clock outside / somewhere else.
Right now the just stay open, even if you click on other menus. This looks kinda weird.

![image](https://github.com/saadeghi/daisyui/assets/20150243/9f552218-cfa7-4531-975e-ee1c2b0c07a5)

https://play.tailwindcss.com/lp8zGbrMAs

**Comment by saadeghi** [comment]
There are 2 methods to use `dropdown` class:
1. [using `<details>` and `<summary>` tags](https://daisyui.com/components/dropdown/#method-1-using-details-and-summary-tags)
  Stays open until gets clicked again. Or you can close it using JS by removing the `open` attribute
2. [using `<label>` and CSS focus](https://daisyui.com/components/dropdown/#method-2-using-label-and-css-focus)
  It opens when focused. closes when focus is lost

Let me know if you have any questions
